### DISCUSSION TITLE
Hide roles

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#7466] [Archive]
Hello, i just need a quick help. I want to hide some roles from the members' directory on my website. For the admin role, i got a script from the theme owner. Butt I need to hide more than one role

```php
// Remove admin from the member directory
function seeko_hide_admins_from_directory( $qs = false, $object = false ) {
$excluded_user = implode( ',', get_users( 'role=administrator&fields=ID' ) );
if ( $object != 'members' && $object != 'friends' ) // hide admin to members & friends
return $qs;
$args = wp_parse_args( $qs );
if( ! empty( $args['user_id'] ) )
return $qs;
if( ! empty( $args['exclude'] ) )
$args['exclude'] = $args['exclude'] . ',' . $excluded_user;
else
$args['exclude'] = $excluded_user;
$qs = build_query( $args );
return $qs;
}
add_action( 'bp_ajax_querystring', 'seeko_hide_admins_from_directory', 20, 2 );
```

That's what I got. But when I try to implement more than one role, it's not working. Can anyone help me with it?

**Comment by rijkvanzanten** [comment]
@Lukeable I have no idea what I'm looking at... What is seeko? Are you using wordpress? How is this related to directus?

**Reply by ghost** [reply]
Seeko is the theme I used for the website. It's a wordpress website.
